### PR TITLE
var -> const

### PR DESCRIPTION
**Pull request by MrBrain295**
#### Summary

var -> const
#### Motivation

It was wrong.
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/messagechannel/messagechannel/index.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ IFrame (using {{domxref("MessagePort.message_event", "onmessage")}}), putting it
 {{domxref("MessageChannel.port1")}} is listened to, to check when the message arrives.
 
 ```js
-var channel = new MessageChannel();
-var para = document.querySelector('p');
+const channel = new MessageChannel();
+const para = document.querySelector('p');
 
-var ifr = document.querySelector('iframe');
+ ifr = document.querySelector('iframe');
```
Comment (wbamberg) [inline files/en-us/web/api/messagechannel/messagechannel/index.md:44]:
```suggestion
const ifr = document.querySelector('iframe');
```

Comment (wbamberg) [inline files/en-us/web/api/messagechannel/messagechannel/index.md:44]:
```suggestion
const ifr = document.querySelector('iframe');
```

**Review by wbamberg — COMMENTED** [review]
So I hate to ask for changes to this when I already did, but ideally we would update the underlying example here, too: https://github.com/mdn/dom-examples/blob/master/channel-messaging-basic/index.html , and then update the page that copies the code.

Also as a meta-comment: while it's great to do var->const, what do you think of updating the pages in slightly bigger batches, rather than one file per PR? I'm not sure what a sweet spot it between lots of tiny PRs and PRs that are too big to review comfortably, but I would say maybe 30 pages at a time?

**Comment by MrBrain295** [comment]
I gonna just make a new PR.

**Pull request closed without merging**